File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    var answer = 0;
    let num1=''
    let num2 = ''
  for (let i = 0; i<num_list.length; i++){
        if(num_list[i]%2==0){
            num1+=num_list[i]
        }
      else{
          num2+=num_list[i]
      }
  }
    return Number(num1) + parseInt(num2)
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    answer1 = ''
    answer2 = ''
        
    for i in range(len(num_list)):
        if num_list[i]%2==0:
            answer1 += str(num_list[i])
        elif num_list[i]%2!=0:
            answer2 += str(num_list[i])
    return int(answer1)+int(answer2)
    
    
    # for i in num_list:
    #     if i%2==0:
    #         answer1 += str(i)
    #     elif i%2!=0:
    #         answer2+= str(i)
    # return int(answer1)+int(answer2)


